src/python/pants/bin/remote_pants_runner.py

```diff
@@ -11,20 +11,142 @@
 from builtins import object, str
 from contextlib import contextmanager
 
-from future.utils import raise_with_traceback
-
-from pants.base.exception_sink import ExceptionSink
+from pants.base.exception_sink import ExceptionSink, GetLogLocationRequest, LogLocation
+from pants.base.exiter import Exiter
 from pants.console.stty_utils import STTYSettings
 from pants.java.nailgun_client import NailgunClient
 from pants.java.nailgun_protocol import NailgunProtocol
 from pants.pantsd.pants_daemon import PantsDaemon
 from pants.util.collections import combined_dict
 from pants.util.dirutil import maybe_read_file
+from pants.util.osutil import safe_kill
 
 
 logger = logging.getLogger(__name__)
 
 
+class RemoteExiter(Exiter):
```

Comment:
this may not be required at all, since server (`pantsd`) never sends anything to the client

Reply:
Indeed! This file has been reverted entirely!